testdata/dnn/download_models.py

```diff
@@ -853,6 +853,11 @@ def get_confirm_token(response):  # in case of large files
         url='https://github.com/AlexeyAB/darknet/releases/download/darknet_yolo_v4_pre/yolov4-tiny.weights',
         sha='d110379b7b86899226b591ad4affc7115f707157',
         filename='yolov4-tiny.weights'),
+    Model(
+        name='ResNet-tiny',  # https://github.com/opencv/opencv/issues/17588
+        url='https://drive.google.com/uc?export=download&confirm=QccG&id=1Sbu9Is_QemC75goe9vy-fdUwKEU84Pvk',
```

Comment:
Do we really need entire model or can have a single layer test?

Reply:
Right now I just work on the reproducer outside of my local environment